autotest/units/001_one_port/049_balancer_pure_l3/services.conf

```diff
@@ -0,0 +1,88 @@
+[
+  {
+    "vip": "10.0.0.16",
+    "proto": "tcp",
+    "scheduler": "rr",
+    "reals": [
```

Comment:
is is too implicit to declare pure L3 balancing service with just absence of `vport` field, should i have some flag for this?

Reply:
Looks good for me